nixos/modules/config/pulseaudio.nix

```diff
@@ -1,17 +1,14 @@
 { config, lib, pkgs, ... }:
-
 with pkgs;
```

Comment:
While we're at it, this ugly `with pkgs;` could also be removed.

Reply:
please look at it again